test/inspector_controller_test.dart

```diff
@@ -6,28 +6,23 @@
 import 'dart:async';
 import 'dart:io';
 
-import 'package:meta/meta.dart';
-import 'package:test/test.dart';
-
 import 'package:devtools/globals.dart';
 import 'package:devtools/inspector/flutter_widget.dart';
 import 'package:devtools/inspector/inspector_controller.dart';
 import 'package:devtools/inspector/inspector_tree.dart';
 import 'package:devtools/inspector/inspector_text_styles.dart' as styles;
 import 'package:devtools/inspector/inspector_service.dart';
-import 'package:devtools/service_manager.dart';
 import 'package:devtools/ui/fake_flutter/fake_flutter.dart';
 import 'package:devtools/ui/flutter_html_shim.dart' as shim;
 import 'package:devtools/ui/icons.dart';
 import 'package:devtools/ui/material_icons.dart';
-import 'package:devtools/vm_service_wrapper.dart';
+import 'package:meta/meta.dart';
```

Comment:
keep packages outside of our project listed before packages from our project.

Reply:
done.